packages/eas-cli/src/commands/update/index.ts

```diff
@@ -112,11 +109,11 @@ export default class UpdatePublish extends EasCommand {
       required: false,
     }),
     republish: Flags.boolean({
-      description: 'Republish an update group',
+      description: 'Republish an update group (deprecated, see republish command)',
```

Comment:
```suggestion
      description: 'Republish an update group',
      deprecated: true,
```
(or feel free to leave stuff in description and add `deprecated` field)

Reply:
The `deprecated` feature from OCLIF is far from optimal. It produces a warning on every command invocation, even when not using the flag. Not a strong opinion, but I would advice against using the deprecated feature.